test/templates/unit_test.tpl

```diff
@@ -128,4 +128,8 @@ jobs:
       if: steps.skip-workflow.outputs.skip-workflow == 'false' && steps.changes.outputs.unit_tests == 'true'
       timeout-minutes: 30
       run: |
+        {{if (eq .Platform "mariadb103")}}
+        # mariadb103
+        export MYSQL_FLAVOR="MariaDB103"
+        {{end}}
```

Comment:
If we decide to go with reading the MySQL version from the instance itself, this won't be required.